client/src/App.js

```diff
@@ -16,6 +18,12 @@ let dialogText;
 let buttonText;
 
 export default function App() {
+  // return (
```

Comment:
In future pull requests, try to actually integrate your components so they work with the existing stuff, or rather, put them where they go.